tests/test_jsonschemas.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import requests
+import json 
+import jsonschema_rs
+from pathlib import Path
+LATEST_CONCRETE= "oedatamodel/latest/v111/datapackage/OEDataModel-concrete-datapackage/OEDataModel-concrete-datapackage.json"
+LATEST_NORMAL = "oedatamodel/latest/v111/datapackage/OEDataModel-normalization-datapackage/OEDataModel-normalization-datapackage.json"
+METADATA_GENERIC = "https://raw.githubusercontent.com/OpenEnergyPlatform/oemetadata/develop/metadata/{}/schema.json"
+
+METADATA_VERSIONS = ["v130", "v140", "v141", "v150", "v151"]
```

Comment:
For your information, versions prior to v141 are no longer supported by most tools.